project.clj

```diff
@@ -12,10 +12,12 @@
                  [org.clojure/clojurescript "1.8.40"]
                  [cljs-ajax "0.5.4"]
                  [com.stuartsierra/component "0.3.1"]
+                 [walmartlabs/system-viz "0.1.1"]
```

Comment:
It should, however the runner is on the project and each profile will try to load it.  Created an issue for it: #7